stammer.py

```diff
@@ -128,21 +154,22 @@ def build_output_video(frames_dir, outframes_dir, best_matches, framerate, outpu
         check=True
     )
 
-def create_output_audio(best_matches, modulator_audio, carrier_frames, modulator_frames, samples_per_frame):
+def create_output_audio(best_matches, coefficients, modulator_audio, carrier_frames, modulator_frames, samples_per_frame):
     output_audio = np.zeros(modulator_audio.shape, dtype=float)
+    def get_carrier(k,c):
+        composite_carrier = None
+        for index, element in enumerate(c):
+            if element == 0:
```

Comment:
This ( `break if element == 0` ) means that the frame at index 0 can still be referenced.  Both the matches and coefficients are zero-padded, meaning an element (i.e., coefficient) with value of zero is at the start of the padding.
`k[index]` on the other hand could be zero as a reference to the first carrier frame.